README.md

```diff
@@ -169,8 +169,9 @@ Example: -r .jobscript.FastQC.*
 
 Please note that depending on your favorite shell, you may need quote your regex. 
 
-### Provide a file with several QBiC IDs
-In order to download datasets from several samples at once, you can provide a simple text file with multiple, line-separated, QBiC IDs and hand it to postman with the `-f` option.
+### Provide a manifest file with several QBiC IDs
+In order to download datasets from several samples at once, you can provide a manifest file consisting of multiple, line-separated, QBiC IDs.
```

Comment:
```suggestion
To download datasets from several samples at once, you can provide a manifest file consisting of multiple, line-separated, QBiC IDs.
```
Suggestion by grammarly

Reply:
Why was this marked as resolved?

Reply:
Because both versions are fine (I guess) and I prefer the original wording